ReactiveFeedback/Feedback.swift

```diff
@@ -80,22 +80,23 @@ public struct Feedback<State, Event> {
     }
 
     /// Creates a Feedback which re-evaluates the given effect every time the
-    /// given predicate passes.
+    /// predicate passes a new occurrence of `State` (i.e. every state change).
     ///
     /// If the previous effect is still alive when a new one is about to start,
-    /// the previous one would automatically be cancelled.
+    /// or if the predicate evaluates to `false`, the previous one would automatically
+    /// be cancelled.
```

Comment:
Might be a good idea to suggest alternatives if the desire is to have the producer keep producing events over time.